src/pages/recommendations.jsx

```
@@ -411,235 +431,285 @@ export default function Recommendations () {
+                      setFocusLocation({ lat: beach.lat, lon: beach.lon });
+                      setSelectedBeachNum(beach.num);
+                      scrollToMap(); // Scroll to map when beach card is clicked
+                    }}
+                    className={`rounded-lg overflow-hidden shadow-lg transition-all duration-300 hover:transform hover:scale-105 
+                        ${
+                          suitability[index] === 'Recommended'
+                            ? 'bg-gradient-to-br from-green-900/40 to-emerald-800/40 border border-green-700/50'
+                            : 'bg-gradient-to-br from-red-900/40 to-rose-800/40 border border-red-700/50'
+                        }
+                        cursor-pointer relative group ${selectedBeachNum === beach.num ? 'ring-2 ring-yellow-400 ring-opacity-70' : ''}`}
+                  >
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)